src/services/swap_service.ts

```diff
@@ -69,13 +77,25 @@ export class SwapService {
             from,
             excludedSources,
             affiliateAddress,
+            apiKey,
+            rfqt,
+            // tslint:disable-next-line:boolean-naming
+            skipValidation,
         } = params;
         const assetSwapperOpts = {
             ...ASSET_SWAPPER_MARKET_ORDERS_OPTS,
             slippagePercentage,
             bridgeSlippage: slippagePercentage,
             gasPrice: providedGasPrice,
             excludedSources, // TODO(dave4506): overrides the excluded sources selected by chainId
+            apiKey,
+            rfqt:
+                rfqt === undefined || from === undefined || from === NULL_ADDRESS
+                    ? undefined
+                    : {
+                          ...rfqt,
+                          takerAddress: isETHSell ? getContractAddressesForChainOrThrow(CHAIN_ID).forwarder : from,
```

Comment:
Would be good to add a comment describing what's happening here

Reply:
Addressed in https://github.com/0xProject/0x-api/commit/03f7673baed642e07d54b95b30c9c59ba1ab9033